modules/metering-debugging.adoc

```diff
@@ -176,7 +178,7 @@ $ oc -n openshift-metering port-forward hdfs-datanode-0 9864 <1>
 
 [id="metering-ansible-operator_{context}"]
 == Metering Ansible Operator
-Metering uses the Ansible Operator to watch and reconcile resources in a cluster environment. When debugging a failed metering installation, it can be helpful to view the Ansible logs or status of your MeteringConfig custom resource.
+Metering uses the Ansible Operator to watch and reconcile resources in a cluster environment. When debugging a failed metering installation, it can be helpful to view the Ansible logs or status of your `MeteringConfig` custom resource.
```

Comment:
This isn't part of the diff, but something to call out that the `Accessing Ansible logs` doesn't contain the sidecar container (and thus no need to specify that `-c/--container` parameter) that was present in the previous metering releases.

Reply:
Something to address in a new PR. 👍